docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst

```diff
@@ -0,0 +1,1206 @@
+
+Next, users should edit the new ``config.yaml`` file to customize it for their machine. On most systems, the following fields need to be updated or added to the appropriate section of the ``config.yaml`` file in order to run the out-of-the-box SRW App case:
+
+.. code-block:: console
+
+   user:
+      MACHINE: hera
+      ACCOUNT: an_account
+   workflow:
+      EXPT_SUBDIR: test_community
+   task_get_extrn_ics:
+      USE_USER_STAGED_EXTRN_FILES: true
+      EXTRN_MDL_SOURCE_BASEDIR_ICS: "/path/to/UFS_SRW_App/develop/input_model_data/<model_type>/<data_type>/<YYYYMMDDHH>"
+   task_get_extrn_lbcs:
+      USE_USER_STAGED_EXTRN_FILES: true
+      EXTRN_MDL_SOURCE_BASEDIR_LBCS: "/path/to/UFS_SRW_App/develop/input_model_data/<model_type>/<data_type>/<YYYYMMDDHH>"
+
+where: 
+   * ``MACHINE`` refers to a valid machine name (see :numref:`Section %s <user>` for options).
+   * ``ACCOUNT`` refers to a valid account name. Not all systems require a valid account name, but most Level 1 & 2 systems do. 
+
+   .. hint::
+
+      To determine an appropriate ACCOUNT field for Level 1 systems, run ``groups``, and it will return a list of projects you have permissions for. Not all of the listed projects/groups have an HPC allocation, but those that do are potentially valid account names. 
+
+   * ``EXPT_SUBDIR`` is changed to an experiment name of the user's choice.
+   * ``/path/to/`` is the path to the SRW App data on the user's machine (see :numref:`Section %s <Data>` for data locations on Level 1 systems). 
+   * ``<model_type>`` refers to a subdirectory containing the experiment data from a particular model. Valid values on Level 1 systems correspond to the valid values for ``EXTRN_MDL_NAME_ICS`` and ``EXTRN_MDL_NAME_LBCS`` (see :numref:`Section %s <basic-get-extrn-ics>` or :numref:`%s <basic-get-extrn-lbcs>` for options). 
+   * ``<data_type>`` refers to one of 3 possible data formats: ``grib2``, ``nemsio``, or ``netcdf``. 
+   * ``<YYYYMMDDHH>`` refers to a subdirectory containing data for the :term:`cycle` date (in YYYYMMDDHH format). 
+
+On platforms where Rocoto and :term:`cron` are available, users can automate resubmission of their experiment workflow by adding the following lines to the ``workflow:`` section of the ``config.yaml`` file:
+
+.. code-block:: console
+
+   USE_CRON_TO_RELAUNCH: true
+   CRON_RELAUNCH_INTVL_MNTS: 3
+
+.. note::
+
+   On Orion, *cron* is only available on the orion-login-1 node, so users will need to work on that node when running *cron* jobs on Orion.
+
+When running with GNU compilers (i.e., if the modulefile used to set up the build environment in :numref:`Section %s <BuildExecutables>` uses a GNU compiler), users must also set ``COMPILER: "gnu"`` in the ``workflow:`` section of the ``config.yaml`` file.
+
+.. note::
+
+   On ``JET``, users should add ``PARTITION_DEFAULT: xjet`` and ``PARTITION_FCST: xjet`` to the ``platform:`` section of the ``config.yaml`` file.
```

Comment:
At some point, Jet was running into errors when "xjet" wasn't specified. I am trying to figure out if this was only when running containers or in all experiments. Unfortunately, we have a few teammates out on vacation, so I think we might have to make the change (if necessary) in a separate PR when I'm able to get a clearer answer.

Reply:
Sounds good, I am like 95% sure it's not necessary outside of containers, but it won't cause any serious problems to have this included it just limits the number of nodes available so will increase queue wait times. Hopefully you can get an answer and just remove this in a future PR once you do.

Reply:
@mkavulich I checked with @natalie-perlin who is updating the SRW App modulefiles, and even outside of containers, experiments often fail on Jet unless xjet is specified.

Reply:
Interesting, I've never run into that myself (or maybe I have and didn't realize what was causing the failure!). @natalie-perlin can you open an issue for this? This seems like something we should get to the root cause of since only using xjet would significantly reduce the number of nodes available for testing, and currently the workflow does *not* contain this partition restriction. 

Since xjet does not have the most memory or the most cores per node (https://jetdocs.rdhpcs.noaa.gov/wiki/index.php/Jet_System_Overview), and the fact that it only *occasionally* fails makes me think that maybe only one or two older partitions is the problem, and maybe it's a memory issue (that could be solved by specifying the necessary memory rather than restricting by partition) or something similar.

Reply:
@mkavulich - please note this is not a bug, but the differences in architecture. xjet is the newer system, and as far as I remember, the only one that has AVX2 vector instructions that are needed for the UFS. The software stack had to be built with no special instructions and AVX2, so some of the SRW tasks could run on other Jet partitions, too (sjet, vjet, kjet). The weather model needs to go to xjet only.